applications/ava-gst-deploy/deploy/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
The Travis is reporting an error since it is not able to find the license headers here. Request you to either add the below lines on top or else remove this file altogether.

# SPDX-License-Identifier: Apache-2.0
# Copyright (c) 2021 InReality LLC